sdk/txbuild/formation.go

```diff
@@ -35,6 +37,13 @@ func Formation(p FormationParams) (*txnbuild.Transaction, error) {
 		SourceAccount: p.InitiatorEscrow.Address(),
 		Signer:        &txnbuild.Signer{Address: p.InitiatorSigner.Address(), Weight: 1},
 	})
+	if !p.Asset.IsNative() {
```

Comment:
the trustlines for escrow accounts is created in the Formation transaction.

see the [SEP setup steps](https://github.com/stellar/experimental-payment-channels/blob/main/specifications/sep-payment-channel-transactions.md#setup) under the formation transaction part